replit_river/session.py

```diff
@@ -267,18 +267,25 @@ async def _heartbeat(
                     ACK_BIT,
                 )
                 self._heartbeat_misses += 1
+                logging.debug(
+                    "heartbeat right now : {} {} {}".format(
+                        self._heartbeat_misses,
+                        self._transport_options.heartbeats_until_dead,
+                        self._transport_options.heartbeat_ms,
+                    )
+                )
                 if (
                     self._heartbeat_misses
                     >= self._transport_options.heartbeats_until_dead
```

Comment:
```suggestion
                    > self._transport_options.heartbeats_until_dead
```

after https://github.com/replit/river/commit/8f796736a54bb09939208637b77b9b055cc30ce4

Reply:
Done